config/locales/en.yml

```diff
@@ -1827,6 +1827,7 @@ en:
   label_project_new: "New project"
   label_project_plural: "Projects"
   label_project_settings: "Project settings"
+  label_project_storage_plural: "Project storages"
```

Comment:
@fraber Where, in which UI, does that string appear? The term "Project storages" does not make sense for normal users or admins. I argue that this label is not of any help for humans. It is a technical name that stems from the join table naming convention of Rails. A better name could be "Activations of file storages per project"... ok, looks long and clumsy but at least it explains better what it is.

CC @apfohl

Reply:
The string appears in /projects/demo-project/copy in the "Copy Options" section. 
It appears together with other options including "Boards", "Forums", "Work package attachments" etc as the options of what objects to copy when creating a copy of a project.

I find it very difficult to come up with a reasonable name. Your "Activations of ..." above clearly doesn't match the stype of the other options, as it is too long.

Saying that, following the style of the other options, a better choice might be just "Storages", as most other options are kept short and "Custom queries" or "Versions" may also not necessarily understood by users. Maybe you want to add a ticket to add help text to the page to explain? I'll change to "Storages".